### PR TITLE
Vue/padv 268

### PR DESCRIPTION
**Pull request by sergivalero20**
PADV-268 - Update CertPREP eComm pages

## Ticket for this PR:

[PADV-268](https://agile-jira.pearson.com/secure/RapidBoard.jspa?rapidView=51807&view=detail&selectedIssue=PADV-268).

## Worked on this PR

- [x] Remove price and align Enroll button (basket view) 
- [x] Remove enrollment number and Quantity Column (receipt view)

## New changes

basket
receipt

## Changes requested by Scott Dunn

Screen Shot 2022-12-19 at 2 57 09 PM
Screen Shot 2022-12-19 at 2 57 19 PM

### CODE REVIEW

**Comment by alexjmpb** [comment]
Did you made changes on all templates that were added? Remember that the first commit has to be solely to add the original templates, and then in the latter commits apply the changes to them so they can be visible.

**Comment by alexjmpb** [comment]
Could you please change the title of the PR to "PADV-268 - Update CertPREP eComm pages"

File: ecommerce/certprep/static/sass/base/_variables.scss [hunk 1]
```diff
@@ -0,0 +1,8 @@
+// Theme variables.
+
+// Colours
```
Comment (alexjmpb) [inline ecommerce/certprep/static/sass/base/_variables.scss]:
```suggestion
// Colors
```

**Comment by sergivalero20** [comment]
> Could you please change the title of the PR to "PADV-268 - Update CertPREP eComm pages"

Title changed.

**Comment by sergivalero20** [comment]
> Did you made changes on all templates that were added? Remember that the first commit has to be solely to add the original templates, and then in the latter commits apply the changes to them so they can be visible.

I copied the folders from ecommerce/themes/red-theme to create CertPREP theme. So, initially, red theme only included static and templates folders. From that moment I commited the changes, and then, I continued adding files from AWS Theme to customized certPREP theme.

File: ecommerce/certprep/static/sass/base/main.scss [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+// views
+// --------------------
+@import 'views/basket';
+@import 'views/credit';
+@import 'views/course_admin';
+@import 'views/coupon_admin';
+@import 'views/coupon_offer';
+@import 'views/error';
+@import 'views/receipt';
```
Comment (alexjmpb) [inline ecommerce/certprep/static/sass/base/main.scss]:
```suggestion
@import 'views/basket';
@import 'views/receipt';
```

What do you think about removing the views that are not being use?

File: ecommerce/certprep/static/sass/base/main.scss [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+// base overrides
+// --------------------
+@import 'base';
+
+// components
+// --------------------
+@import 'components/buttons';
+@import 'components/navbar';
+@import 'components/footer';
```
Comment (alexjmpb) [inline ecommerce/certprep/static/sass/base/main.scss]:
```suggestion
@import 'components/navbar';
```

File: ecommerce/certprep/static/sass/base/main.scss [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+// --------------------
+@import '../../bower_components/bootstrap-sass/assets/stylesheets/bootstrap';
+@import '../../bower_components/bootstrapaccessibilityplugin/plugins/css/bootstrap-accessibility';
+@import '../../bower_components/datatables/media/css/dataTables.bootstrap';
+@import '../../bower_components/fontawesome/scss/font-awesome';
+@import '../../vendor/dataTables/dataTables.fontAwesome';
+
+// pattern library
+// --------------------
+@import '_pattern_library';
```
Comment (alexjmpb) [inline ecommerce/certprep/static/sass/base/main.scss]: Is all of this necessary?

**Pull request closed without merging**